src/containerapp/azext_containerapp/_help.py

```diff
@@ -756,16 +284,6 @@
               --image imageName
 """
 
-helps['containerapp job update'] = """
```

Comment:
For job related commands, all commands are moved to cli core? What about job create? It is left in extension because there is new feature?

Reply:
For `az containerapp job create`, we support `--environment-type`, which is preview feature. Other commands are moved to cli core.